src/index.css

```diff
@@ -225,3 +224,24 @@ body {
   justify-content: center;
   align-items: center;
 }
+
+.percentageCircleContainer {
+  position: relative;
+  width: 100px;
+  height: 100px;
+  margin-right: 0.8rem;
+}
+
```

Comment:
- [ ] It would be better if we remove the book id in the UI since it is not part of the required design

Reply:
It's not an improvement for the design, since I was following the styles for that text. The problem is that the API for the books doesn't work with the author's name, only the book id. It's nonsense to display a generic book author everytime but I did it anyway, next time think of this as a unique identifier for a book, because library's do provide that type of info for books on the platform.